Persistant-Storage-Snapshot/DeserializationEx.java

```diff
@@ -0,0 +1,35 @@
+// where should this go? same folder as ServerMain? as methods in Snapshotter or ServerMain?
```

Comment:
ServerMain is going to deserialize on startup, so I would put it in the same folder it is in.